src/hooks/govwallet/useGovWalletBalance/useGovWalletBalance.tsx

```diff
@@ -73,8 +73,12 @@ export const useGovWalletBalance = (
 
           // We only check the eligibility of the balance of the first account
           const areAllBalancesEligible = results.every(
-            // Check if balance is strictly equals to 10000 cents
-            ({ accountDetails }) => accountDetails[0].balance === 10000
+            // if govwalletExactBalanceValue == -1 or undefined, dont check exact value
+            ({ accountDetails }) =>
+              accountDetails[0].balance ===
+                features?.govwalletExactBalanceValue ||
+              features?.govwalletExactBalanceValue === -1 ||
+              features?.govwalletExactBalanceValue === undefined
```

Comment:
Any way to simplify this statement, e.g using intermediate variables to document what the -1 and undefine mean? What's the difference between -1 and undefined?

Reply:
thanks updated to make clearer